### PR TITLE
feat: add UI libs

### PR DESCRIPTION
**Pull request by mozwell**
# What's New?
- Add & configure tailwindCSS
- Add & configure daisyUI
- Add & configure icon-park
- Add clsx
- Button component done

### CODE REVIEW

File: components/button.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    'btn-sm': size === 'sm',
+    'btn-xs': size === 'xs',
+    'gap-2': startIcon,
+    loading: loading,
+  })
+
+  return (
+    <button className={cls} disabled={disabled}>
+      {!loading && startIcon}
+      {loading ? 'loading' : children}
```
Comment (renzholy) [inline components/button.tsx:32]: why don't use spinner for loading?
https://daisyui.com/components/button/#button-with-loading-spinner

File: yarn.lock [hunk 2]
```diff
@@ -549,6 +549,11 @@
   resolved "https://registry.yarnpkg.com/@humanwhocodes/object-schema/-/object-schema-1.2.1.tgz#b520529ec21d8e5945a1851dfd1c32e94e39ff45"
   integrity sha512-ZnQMnLV4e7hDlUvw8H+U8ASL02SS2Gn6+9Ac3wGGLIe7+je2AeAOxPY+izIPJDfFDb7eDjev0Us8MO1iFRN8hA==
 
+"@icon-park/react@^1.4.2":
+  version "1.4.2"
+  resolved "https://registry.npmmirror.com/@icon-park/react/-/react-1.4.2.tgz#db761ebc0e685702b80c4aaf5b59af5ccda40ecc"
```
Comment (renzholy) [inline yarn.lock:554]: do not use npmmirror, use yarnpkg instead

File: components/button.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  const cls = clsx({
+    btn: true,
+    'btn-block': fullWidth,
+    'btn-outline': variant === 'outline',
+    'btn-lg': size === 'lg',
+    'btn-sm': size === 'sm',
+    'btn-xs': size === 'xs',
+    'gap-2': startIcon,
+    loading: loading,
```
Comment (renzholy) [inline components/button.tsx:26]: loading,

File: components/button.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  const { disabled, size, variant, fullWidth, startIcon, loading, children } =
+    props
+
+  const cls = clsx({
+    btn: true,
+    'btn-block': fullWidth,
+    'btn-outline': variant === 'outline',
+    'btn-lg': size === 'lg',
+    'btn-sm': size === 'sm',
+    'btn-xs': size === 'xs',
```
Comment (renzholy) [inline components/button.tsx:24]:
```js
[`btn-${size}`]: true
```

File: pages/_app.tsx [hunk 5]
```diff
@@ -22,13 +24,18 @@ const wagmiClient = createClient({
   provider,
 })
 
+// We use 1.5rem for all SVG icons to ensure compatibility with daisyUI
+const IconConfig = { ...DEFAULT_ICON_CONFIGS, size: '1.5rem' }
```
Comment (renzholy) [inline pages/_app.tsx:28]:
```
iconConfig
```

File: components/button.tsx (hunk 1 above)
Comment (mozwell) [reply components/button.tsx]: We agreed to use react-daisyui, so this component is deleted.

File: yarn.lock (hunk 2 above)
Comment (mozwell) [reply yarn.lock]: Fixed.

File: components/button.tsx (hunk 3 above)
Comment (mozwell) [reply components/button.tsx]: We agreed to use react-daisyui, so this component is deleted.

File: components/button.tsx (hunk 4 above)
Comment (mozwell) [reply components/button.tsx]: We agreed to use react-daisyui, so this component is deleted.

File: pages/_app.tsx (hunk 5 above)
Comment (mozwell) [reply pages/_app.tsx]: FIxed.

**Review by renzholy — APPROVED** [review]

**Pull request merged**